### ISSUE TITLE
Write up paper on this project.

### ISSUE THREAD

**Issue by wfastrononomer** [wfau/ScienceArchives#197]
Write paper on the design of the all sky NIR catalogue.
GitHub / Overleaf project.

**Comment by wfastrononomer** [comment]
Started GitHub/Overleaf project.

**Comment by wfastrononomer** [comment]
Example usage needs to be documented in the paper but also in the Archive webpage.

**Comment by tms-epcc** [comment]
29/APR/25
- on hold for the moment

**Comment by tms-epcc** [comment]
14/MAY/25
- @wfastrononomer and @harry-rendell will start to consider what to include in the paper

**Comment by wfastrononomer** [comment]
Overleaf:
https://www.overleaf.com/project/61b0e37e514480494aefd300

**Comment by tms-epcc** [comment]
01/OCT/25
- an initial iteration will refer to the Internal Data Release (IDR)
- will share this initial iteration with external authors

**Comment by tms-epcc** [comment]
22/JAN/26
waiting for IDR before starting

**Comment by tms-epcc** [comment]
20/MAY/26
- in progress 
- producing necessary plots 
- not yet shareable

**Comment by tms-epcc** [comment]
24/JUN/26
- slow progress, making plots (doing these can be helpful for the checks)

**Comment by wfastrononomer** [comment]
Added in histograms of n(epochs), seeing and AB maglim.

**Comment by tms-epcc** [comment]
05/AUG/26
- some progress on writing but no yet ready to share
